sql/vacuum/vacuum.go

```diff
@@ -0,0 +1,29 @@
+package vacuum
+
+import (
+	"fmt"
+
+	"github.com/spacemeshos/go-spacemesh/sql"
+)
+
+func VacuumDB(db sql.Executor) error {
+	if _, err := db.Exec("vacuum", nil, nil); err != nil {
+		return fmt.Errorf("vacuum %w", err)
+	}
+	if _, err := db.Exec("pragma wal_checkpoint(TRUNCATE)", nil, nil); err != nil {
+		return fmt.Errorf("wal checkpoint %w", err)
+	}
+	return nil
+}
+
+func FreePct(db sql.Executor) (int, error) {
```

Comment:
could you please use full word? there is no reason to decipher this

Reply:
deleted.